### DISCUSSION TITLE
How is the link structured for a ZTS download?

### DISCUSSION THREAD

**Discussion by shyim** [cachewerk/relay#133] [Support] [answered]
The manual section does not have any information how the ZTS url should be.

https://relay.so/docs/1.x/installation#manual-installation

I tried by appending zts, but did not worked like here https://github.com/cachewerk/relay/blob/main/docker/debian/debian10-zts.Dockerfile#L33

**Comment by michael-grunder** [comment]
We've got a little builds selection page [here](https://relay.so/builds)

Here are the `zts` links at least for debian.

https://builds.r2.relay.so/v0.7.0/relay-v0.7.0-php7.4-debian-x86-64+zts.tar.gz
https://builds.r2.relay.so/v0.7.0/relay-v0.7.0-php8.0-debian-x86-64+zts.tar.gz
https://builds.r2.relay.so/v0.7.0/relay-v0.7.0-php8.1-debian-x86-64+zts.tar.gz
https://builds.r2.relay.so/v0.7.0/relay-v0.7.0-php8.2-debian-x86-64+zts.tar.gz

Hopefully I understand you're question correctly :smile:

**Reply by shyim** [reply]
Now I got the real problem. there is no PHP 8.3 ZTS.  😂  with 8.2 it works perfectly fine the link

**Reply by tillkruss** [reply]
You're right. I'm not sure why those were skipped. I'll kick off the builds.

**Reply by tillkruss** [reply]
@shyim: What exact OS are you running?

**Reply by szepeviktor** [reply]
A published JSON file with all supported Relay/OS/PHP versions would solve all these release problems.

**Reply by shyim** [reply]
glibc based os, debian x86 and aarch64 with zts would be fine :)

**Reply by tillkruss** [reply]
@shyim: We used `php:8.x-zts-buster` from DockerHub, but 8.3 isn't available for `buster`.

I think going directly to `8.3-zts-bookworm` would be good because of the OpenSSL v3 switch.

**Reply by shyim** [reply]
should be fine :)

**Reply by shyim** [reply]
You can drop me also the .so before, I can try it out

**Reply by tillkruss** [reply]
We decided against building newer artifacts for ZTS because until we get a paying customer for it. It's too much overhead and we don't know anybody using this in production.

**Reply by tillkruss** [reply, marked as answer]
@shyim We're adding some `relay-zts.so` binaries to the artifacts instead of dedicated URLs.
